src/org/mozilla/javascript/Sorting.java

```diff
@@ -85,10 +97,28 @@ private static int log2(int n)
         return (int)(Math.log10(n) / Math.log10(2.0));
     }
 
-    private static int median(int n1, int n2, int n3)
+    /*
+    Return the index of the smallest of three elements in the specified array range -- the
+    first, the last, and the one in the middle.
```

Comment:
Is this correct? From my reading `median()` returns the index of the median of three elements (start, end and in the middle), not the smallest in range.